content/docs/instrumenting/writing_clientlibs.md

```diff
@@ -327,12 +327,7 @@ readable formats) if it can be implemented without a significant resource cost.
 ## Standard and runtime collectors
 
 Client libraries SHOULD offer what they can of the Standard exports, documented
-at
-[https://docs.google.com/document/d/1Q0MXWdwp1mdXCzNRak6bW5LLVylVRXhdi7_21Sg15xQ/edit](https://docs.google.com/document/d/1Q0MXWdwp1mdXCzNRak6bW5LLVylVRXhdi7_21Sg15xQ/edit)
-
-In addition, client libraries are ENCOURAGED to also offer whatever makes sense
-in terms of metrics for their language’s runtime (e.g. Garbage collection
-stats).
+at [standard exports](/docs/instrumenting/standard_exports).
```

Comment:
This was mostly to separate it from clients.

If some things are standardized, it's also relevant for scrapers and consoles, not just for the ones writing clients.

Say I wanna write a generic template for metric visualizations, it'd be nice to know what some standard metrics are named, but I don't care about what conventions for other metrics are, I just want to be able to scrape some default ones.